drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-businesscentral-client/src/test/java/org/drools/workbench/screens/scenariosimulation/businesscentral/client/rightpanel/coverage/CoverageReportPresenterTest.java

```diff
@@ -217,11 +217,11 @@ public void populateScenarioList() {
         List<Map<String, Integer>> resultCounters = resultCounterCaptor.getAllValues();
         assertEquals(scenarioCounterLocal.size(), resultCounters.size());
         assertTrue(resultCounters.get(0).keySet().contains("d1"));
-        assertEquals(1, resultCounters.get(0).get("d1"));
+        assertEquals(1, (int) resultCounters.get(0).get("d1"));
         assertTrue(resultCounters.get(0).keySet().contains("d2"));
-        assertEquals(1, resultCounters.get(0).get("d2"));
+        assertEquals(1, (int) resultCounters.get(0).get("d2"));
         assertTrue(resultCounters.get(1).keySet().contains("d2"));
-        assertEquals(1, resultCounters.get(1).get("d2"));
+        assertEquals(1, (int) resultCounters.get(1).get("d2"));
     }
 
     @Test
```

Comment:
Cannot find the test where   presenterSpy.downloadReportCommand = execute
But found for negative case: 
    @Test
    public void onDownloadButtonClicked_NoCommandAssigned() {
        presenterSpy.downloadReportCommand = null;
        verify(downloadReportCommandMock, never()).execute();
    }

Reply:
Could we add assertTrue(downloadReportButtonMock.disabled); to this above test as well?

Reply:
@dupliaka done